__config__/webpack/webpack.config.js

```diff
@@ -7,57 +7,76 @@ const { smart: merge } = require('webpack-merge');
 
 const dist = path.resolve(__dirname, '../../dist');
 
-const config = platform => ({
-    name: platform,
-    entry: {
-        app: [
-            'preact-emotion', // force preact-emotion into the vendor chunk
-            `./src/${platform}.js`,
+const config = ({ platform, pages }) => {
+    const entry =
+        platform === 'server'
+            ? {
+                  app: [
+                      'preact-emotion', // force preact-emotion into the vendor chunk
+                      `./src/${platform}.js`,
```

Comment:
this could just be `./src/server.js` now?

Reply:
Yeah, could do. This is me being too DRY!